lib/galaxy/managers/groups.py

```diff
@@ -119,25 +112,17 @@ def _get_group(self, sa_session: galaxy_scoped_session, group_id: int) -> model.
             raise ObjectNotFound("Group with the provided id was not found.")
         return group
 
-    def _get_users_by_encoded_ids(
-        self, sa_session: galaxy_scoped_session, encoded_user_ids: List[EncodedDatabaseIdField]
+    def _get_users_by_ids(
+        self, sa_session: galaxy_scoped_session, user_ids: List[DecodedDatabaseIdField]
     ) -> List[model.User]:
-        user_ids = self._decode_ids(encoded_user_ids)
         return get_users_by_ids(sa_session, user_ids)
```

Comment:
Looks like this function is no longer relevant and can be replaced directly by `get_users_by_ids`

Reply:
Thank you for the review. I will apply them ASAP.